packages/ffe-form-react/src/StepSelector.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import RadioButtonInputGroup from './RadioButtonInputGroup';
+import RadioButton from './RadioButton';
+import { arrayOf, string, shape, func, number } from 'prop-types';
+
+const StepSelector = ({
+    name,
+    options,
+    initialSelected,
+    label,
+    description,
+    onChange,
+}) => {
+    const initialValue = initialSelected ? Number(initialSelected.value) : null;
+    const [selectedValue, setSelectedValue] = useState(initialValue);
+
+    const optionSelected = value => {
+        const formattedValue = Number(value);
+        setSelectedValue(formattedValue);
+        onChange(formattedValue);
+    };
+
+    return (
+        <RadioButtonInputGroup
+            inline={true}
+            label={label}
+            description={description}
+            name={name}
+            onChange={e => optionSelected(e.target.value)}
+            selectedValue={selectedValue}
+        >
+            {inputProps => (
+                <div
+                    className="ffe-step-selector"
+                    id="ffe-step-selector-container"
+                    role="ffe-step-selector-container"
+                >
+                    {options.map((option, i) => {
+                        const isSelected = option.value === selectedValue;
+                        const optionClassName = classNames(
+                            'ffe-step-selector-option',
+                            'ffe-step-selector-option--text-center',
+                            { 'ffe-step-selector-option-selected': isSelected },
+                        );
+                        return (
+                            <RadioButton
+                                {...inputProps}
+                                name={name}
+                                key={option.value}
+                                value={option.value}
+                                className={optionClassName}
+                                tabIndex={i}
```

Comment:
tabIndex={i} er ingetin man setter for elementer som allrede førljer naturligt html/retning

Det er kun om du vill endra naturlig flyt.

Radio knappar funker feks heller ikke som lenker når man tabbar. Tab tar deg til førsta.... sedan velger du neste med pilarna. Det er slik html fungerar og det er inget vi skall endre.